internal/access/identity.go

```diff
@@ -64,7 +64,7 @@ func DeleteIdentity(c *gin.Context, id uid.ID) error {
 		return fmt.Errorf("delete identity access keys: %w", err)
 	}
 
-	groups, err := data.ListGroups(db, nil, data.ByGroupMember(id))
+	groups, err := data.ListGroups(db, data.ListGroupsOptions{ByGroupMember: id})
```

Comment:
This was a required selector, but there's no way this can be called with a zero value for `id`, because the API request validation marks it as required.